t/apicast-policy-rate-limit.t

```diff
@@ -1,9 +1,15 @@
 use lib 't';
 use Test::APIcast 'no_plan';
+use Cwd qw(abs_path);
 
+$ENV{TEST_NGINX_LUA_PATH} = "$Test::APIcast::spec/?.lua;$ENV{TEST_NGINX_LUA_PATH}";
```

Comment:
Why is this needed?

Reply:
In order to resolve `'fixtures.rsa'`.
I referenced [apicast-oidc.t](https://github.com/3scale/apicast/blob/master/t/apicast-oidc.t).

Reply:
I see. I think we can find a better solution. I think these tests should make use of the `fixtures` directory in `t` instead of the one in `spec`. However, as you said, we're doing the same in `apicast-oidc.t`. We should fix this, but I think we can do it after this PR.